vcd/resource_vcd_vapp_org_network.go

```diff
@@ -300,7 +300,7 @@ func resourceVcdVappOrgNetworkImport(d *schema.ResourceData, meta interface{}) (
 	}
 
 	if vappNetworkToImport.Configuration.FenceMode == types.FenceModeIsolated ||
-		(vappNetworkToImport.Configuration.FenceMode == types.FenceModeNAT && vappNetworkToImport.Configuration.Features.DhcpService != nil) {
+		(vappNetworkToImport.Configuration.FenceMode == types.FenceModeNAT && vappNetworkToImport.Configuration.IPScopes.IPScope[0].IsInherited == false) {
```

Comment:
We should check for `vappNetworkToImport.Configuration.IPScopes` to be not nil , and for `vappNetworkToImport.Configuration.IPScopes.IPScope` to have elements

Reply:
Also wondering what is the impact (or maybe none) of `NetworkConfiguration.ParentNetwork`

Reply:
it always return, but added